bin/release-build.sh

```diff
@@ -12,5 +12,10 @@ if [[ $# != 1 ]]; then
 fi
 
 log_file="/build/logs/${1}.log"
-nohup time ruby ${BUILD_DIR}/scripts/vmbuild.rb --type release --upload --reference $1 --copy-dir ${1%%-*} > $log_file 2>&1 &
+copy_dir="${1%%-*}"
+
+nohup sh -c \
+        "echo time ruby ${BUILD_DIR}/scripts/vmbuild.rb --type release --upload --reference $1 --copy-dir $copy_dir \
+         && time ruby ${BUILD_DIR}/scripts/vmpublish.rb --copy-dir $copy_dir" \
```

Comment:
I moved it to a separate segment on purpose. Happy to reconsider that but let me explain my reasoning here first to see if it make sense.

These were my thoughts:

1. There can be multiple locations to which you upload a single image. Once we support AWS we will get into this situation, where we have to upload the same image to multiple regions. We'd have to have a nested loop in the build loop and I though that would be more complicated and make things really slow.

2. It allows some sanity tests before publishing. I  In my view "publish" is a more impactful operation than "upload". The upload puts stuff on our file server from which people can download it. But "publish" will immediate make the image the live default choice for users of those services. For example, if you use "vagrant init manageiq/darga" that would immediately take the latest version by default. Same for the cloud launcher on GCE (we are working on this now). So doing some quick sanity testing on the build before a publish would not be a bad thing.

(Note that this means we shouldn't call the publish script unconditionally from the "release-build.sh" script. That was an oversight.)

3. Separation makes things more reliable. It allows the build phase to fail, without having some publish locations updated and others not. And it also allows the publish phase to be idempotent. If the publish fails, simply re-run the command. It won't build anything, but re-upload the most recently built set of images. The way the script works, it will overwrite anything with the specified version that is already there.